public/app/features/explore/PrometheusListView/RawListContainer.tsx

```diff
@@ -0,0 +1,142 @@
+import { css } from '@emotion/css';
+import { cloneDeep } from 'lodash';
+import React, { useEffect, useRef, useState } from 'react';
+import { useWindowSize } from 'react-use';
+import { VariableSizeList as List } from 'react-window';
+
+import { DataFrame, Field } from '@grafana/data/src';
+import { Button, stylesFactory } from '@grafana/ui/src';
+
+import { getRawPrometheusListItemsFromDataFrame } from '../utils/getRawPrometheusListItemsFromDataFrame';
+
+import { ItemLabels } from './ItemLabels';
+import RawListItem from './RawListItem';
+
+export type instantQueryRawVirtualizedListData = { Value: string; __name__: string; [index: string]: string };
+
+export interface RawListContainerProps {
+  tableResult: DataFrame;
+}
+
+const getRawListContainerStyles = stylesFactory(() => {
+  return {
+    wrapper: css`
+      height: 100%;
+      overflow: scroll;
+    `,
+    mobileWrapper: css`
+      height: 100%;
+      overflow: scroll;
+      .list-item-attribute {
```

Comment:
I don't like this extra specificity we're adding here with the second class, is there another solution?

Reply:
Yeah I moved the prop into the component so it could handle this style